db/migrations/20170817142615_create_movies.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
```

Comment:
This is not your fault since we hadn't updated all of Miyagi yet but we've moved to using arrow functions whenever possible, including in migrations. Both the `up()` and `down()` functions can be converted to use ES6 arrow functions in this case. 

I've gone ahead and updated Miyagi so this is not a problem in the future but this is something we can fix here 😄